docs/subsystems/settings.md

```diff
@@ -57,7 +57,10 @@ In a production environment, we have:
   secrets (e.g. passwords, secret API keys, cryptographic keys, etc.).
   The way we generally do settings that can be controlled with shell
   access to a Zulip server is to put a default in
-  `zproject/default_settings.py`, and then override it here.
+  `zproject/default_settings.py`, and then override it here.  As this
+  is the main documentation for Zulip settings, we recommend [carefully
+  updating your `/etc/zulip/settings.py` every major
+  release](../production/upgrade-or-modify.html#updating-configuration-files).
```

Comment:
Should we frame this as an explanation for Zulip developers about this recommendation we make to production installations (i.e. not using "your" here)?

Reply:
:nod:  Makes sense, since it's a different audience than the other locations.